### PR TITLE
Event Hooks Overview: Updates based on comments

### PR DESCRIPTION
**Pull request by motikagan-okta**
## Description:
- Conceptual overview of Event Hooks.
- Event Hooks functionality is now targeted for EA release in 2019.04.1.
- There was a previous pull request for this docs page, https://github.com/okta/okta-developer-docs/pull/168, however it cannot be re-opened, therefore starting this new one.
- Edits in this pull request are based on the comments on the previous pull request.

### Resolves:

* [OKTA-209169](https://oktainc.atlassian.net/browse/OKTA-209169)

### CODE REVIEW

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+To secure the communication channel between Okta and your external service, HTTPS is used for requests, and support is provided for header-based authentication. Okta recommends that you implement an authentication scheme using the authentication header, to be used to authenticate every request received by your external service.
+
+### Your Service's Responses to Event Delivery Requests
+
+Your external service's responses to Okta's ongoing event delivery POST requests should all be empty, and should have an HTTP status code of 200 (OK) or 204 (No Content). As a best practice, you should return the HTTP response immediately, rather than waiting for any of your own internal process flows triggered by the event to complete.
+
+### Rate Limits
+
+Event hooks are limited to sending 100 000 events per 24-hour period.
```
Comment (ankurdatta-okta) [inline packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: Minor typo -- there is an extra space in 100,000.

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+To handle event hook calls from Okta, you need to implement a web service with an Internet-accessible endpoint. It's your responsibility to develop the code and to arrange its hosting on a system external to Okta. Okta defines the REST API contract for the requests that it will send.
+
+Event hooks are related to, but different from, Okta [inline hooks](/use_cases/inline_hooks/). Event hooks are meant to provide information about events that occurred, not offer a way to affect the underlying Okta process flow. For that, you need to use inline hooks. Also, unlike inline hooks, event hooks are asynchronous calls, meaning that the process flow that triggered the event hook continues without stopping or waiting for any response from your external service.
+
+You can create a maximum of ten event hooks in your org. Each event hook can be configured to deliver multiple event types.
```
Comment (ankurdatta-okta) [inline packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: Please mention that the limit is 10 _Active_ event hooks.  An org can have, for example, 10 active hooks and 100 inactive hooks.

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+## Sample Event Delivery Payload
+
+The following is an example of a JSON payload of a request from Okta to your external service:
+
+```json
+{
+  "eventType": "com.okta.event_hook",
+  "eventTypeVersion": "1.0",
+  "cloudEventsVersion": "0.1",
+  "eventID": "b5a188b9-5ece-4636-b041-482ffda96311",
```
Comment (amoghvasekar-okta) [inline packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: nit : We had some minor updates after our last conversation around this.
Changed `eventID` to `eventId`.
Added new top-level field ` "contentType": "application/json"`

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md (hunk 2 above)
Comment (amoghvasekar-okta) [reply packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: ^^ +1. Something like - `You may have at most 10 active and verified event hooks in your org at any point`.

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md (hunk 1 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: Changed to "one hundred thousand", in words, for clarity.

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md (hunk 2 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: Changed to: "You can have a maximum of 10 active and verified event hooks in your org at any point."

File: packages/@okta/vuepress-site/use_cases/event_hooks/index.md (hunk 3 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/use_cases/event_hooks/index.md]: Changed capitalization in `eventId` and added `contentType` field.

**Pull request merged**